src/Lexer.php

```diff
@@ -352,4 +375,24 @@ public static function decodeJson($json)
     {
         return json_decode($json, true);
     }
+
+    /**
+     * Escape plain text output before mixing in a html context.
+     * 
+     * This should be used on any input or attributes in a delta operation.
+     * For escaping input, use Line->getInput() instead as it keeps track of only doing it once for nested inline listeners.
+     * 
+     * @since 1.2.0
+     * 
+     * @param  string $value
+     * @return string
+     */
+    public function escape($value)
+    {
+        if ($this->escapeInput === false) {
```

Comment:
Usually we do 

```php
if (!$this->escapedInput)
```

instead of strict compare, but we can keep it if you like.

Reply:
I'm used to strict compare. But I think consistency is more important :)